dynamic_instrumentation/src/main.rs

```diff
@@ -63,6 +63,17 @@ struct Args {
     cargo_args: Vec<OsString>,
 }
 
+/// `cargo` args that we intercept.
+#[derive(Debug, Parser)]
+struct InterceptedCargoArgs {
+    #[clap(long, value_parser)]
+    manifest_path: Option<PathBuf>,
+
+    /// Need this so `--` is allowed.
+    /// Not actually used.
+    extra_args: Vec<OsString>,
+}
```

Comment:
I actually need to fix this to allow arbitrary `cargo` args to pass through without error.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/627/commits/561a2701d7a29e45314411a16a2933f9641d1fb5.